doc/connectivity/bluetooth/api/mesh/access.rst

```diff
@@ -190,7 +190,6 @@ Composition Data Page 1 provides information about the relationships between mod
 and is mandatory for all mesh devices. A model may extend and/or correspond to one
 or more models. A model can extend another model by calling :c:func:`bt_mesh_model_extend`,
 or correspond to another model by calling :c:func:`bt_mesh_model_correspond`.
-:kconfig:option:`CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE` specifies how many model
```

Comment:
Why did you remove mentioning of `CONFIG_BT_MESH_MODEL_EXTENSION_LIST_SIZE`?

Reply:
Accidentally. (and got approvals).